src/RoslynInsertionTool/scripts/VS-Insertion.ps1

```diff
@@ -43,10 +43,9 @@ function Do-Insertion(
 ### F# Insertions (handled by the F# team)
 ###
 
-#Do-Insertion -component "F#"                -queueName "FSharp-Signed"         -fromBranch "dev15.8"           -toBranch "rel/d15.8" -insertCore "false" -insertDevdiv "false" -queueValidation "true" -dropPath "\\cpvsbuild\drops\FSharp"
-Do-Insertion -component "F#"                -queueName "FSharp-Signed"         -fromBranch "dev15.9"           -toBranch "rel/d15.9" -insertCore "false" -insertDevdiv "false" -queueValidation "true" -dropPath "\\cpvsbuild\drops\FSharp"
-#Do-Insertion -component "F#"                -queueName "FSharp-Signed"         -fromBranch "dev16.0"        -toBranch "lab/ml" -insertCore "false"    -insertDevdiv "false" -dropPath "\\cpvsbuild\drops\FSharp"
-#Do-Insertion -component "F#"                -queueName "FSharp-Signed"         -fromBranch "dev16.0"        -toBranch "lab/ml" -insertCore "false"    -insertDevdiv "false" -dropPath "\\cpvsbuild\drops\FSharp"
+Do-Insertion -component "F#"                -queueName "FSharp-Signed"         -fromBranch "dev15.8"        -toBranch "rel/d15.8"    -insertCore "false" -insertDevdiv "false" -queueValidation "true" -dropPath "\\cpvsbuild\drops\FSharp"
```

Comment:
I don't think we need 15.8 enabled.

Reply:
Oh, I see you saying it will be a no-op. Still, I don't think we need it? Having it there commented out (like the non-VS ones saying it's for servicing) is nice for me, but we can do whatever.

Reply:
If you want it, then no problem as long as it isn't creating unnecessary insertion PRs. I just don't really want to change the non-F# ones if they would cause irrelevant insertion PRs as well. I think if we're going to service 15.8 then someone will come talk to us directly. :)